pages/aduan-warga/detail.vue

```diff
@@ -69,7 +69,7 @@ export default {
           time: '18:00',
           name: 'Admin'
         }
-      ],
+      ].reverse(),
```

Comment:
hallo bli @Ibwedagama 

for this dummy data i make reverse, because in figma milestone showing data from the newst data update to oldst data by date and me make data dummy otherwise, so i did reverse data dummy

![image](https://user-images.githubusercontent.com/58242304/225505325-a051f7e5-6a9a-4aa6-8f41-c3e9148eca90.png)

I don't know what real of data will be provided later by the API

Reply:
ah I see, I think we need to make sure the response data from API is already being sorted so we don't have to sort them manually @rayfajars

Reply:
yes bli @Ibwedagama  thanks for suggest